asmcomp/schedgen.ml

```diff
@@ -383,7 +383,7 @@ method schedule_fundecl f =
       self#reschedule ready_queue 0 (schedule i try_nesting)
     end in
 
-  if f.fun_fast then begin
+  if f.fun_fast && not !Clflags.no_insn_sched then begin
```

Comment:
I don't much like the double negation. You should introduce a second option, `-insn-sched`, document it as the default, and invert the meaning of the `Clflags` variable. Experience shows it's always a good idea to have both sides of a toggle.